source/layout.js

```diff
@@ -32,6 +32,14 @@ const types = [
 ]
 
 const validTypeValues = types.reduce((xs, { values }) => xs.concat(values), [])
+const POSITIONS = Object.freeze(
+  Object.assign(
```

Comment:
I don't know what your browser compatibility guarantees are but `Object.assign` is _not_ compatible with `IE` and it doesn't look like this gets polyfilled when building locally (at least for me). 

Happy to add a polyfill for this or to build the object without using `Object.assign`.